packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -39,19 +39,33 @@ import { useContextMenu } from './contextMenuProvider';
 
 const MenuWrapper = styled.div(
   ({ theme }) => css`
-    padding: ${({ $isIconMenu }) => ($isIconMenu ? '4px 3px' : '8px 0')};
-    background-color: ${theme.colors.bg.primary};
-    border-radius: ${theme.borders.radius.small};
+    background-color: ${({ $isSecondary }) =>
+      $isSecondary ? theme.colors.bg.secondary : theme.colors.bg.primary};
+    border-radius: ${theme.borders.radius.medium};
```

Comment:
Should it be still `small` or [Shadow border radius](https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/design-system/src/components/contextMenu/styled.js#L42) should be changed to `medium`?

Otherwise:
![image](https://user-images.githubusercontent.com/21036224/152334477-87aa7a4b-3be5-40d0-b428-e392fb178316.png)

Reply:
Hm, I'm not seeing this? But perhaps I changed things after you reviewed:

![Screen Shot 2022-02-04 at 10 12 12](https://user-images.githubusercontent.com/637548/152553599-7acd0670-1174-4065-ac9b-f7991b6ae80a.png)

The border-radius on the shadow is [already flexible depending on the menu orientation](https://github.com/GoogleForCreators/web-stories-wp/pull/10438/files#diff-e98500a7b8fdf0a25ebba0b1723fcbe6d5ca8d9afedb0773d1cd4bad29b9078bR42-R43).

Reply:
i just pulled down the latest and can confirm that this issue is resolved.

Reply:
Yep, fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/10438/commits/87aa7bcc20470a917bcd1819f28bf1fd3db6c417